CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [2.1.0] - 2019-12-06
+
+### Changed
+
+- Drop PHP 7.0 Support  - [#267](https://github.com/owncloud/password_policy/issues/267)
+- Use CHAR_SYMBOLS to generate password when no special chars are specified - [#233](https://github.com/owncloud/password_policy/issues/233)
+- Drop PHP 5.6 Support - [#211](https://github.com/owncloud/password_policy/issues/211)
+- Implement phpstan and phan - [#169](https://github.com/owncloud/password_policy/issues/169)
```

Comment:
This does not really matter to stay here. But it is just implementing code quality tools, nothing that directly effects the released app.